app/Models/Forum/Authorize.php

```diff
@@ -16,9 +16,12 @@
  */
 class Authorize extends Model
 {
-    protected $table = 'phpbb_acl_groups';
+    public $incrementing = false;
     public $timestamps = false;
+
+    protected $primaryKey = ':composite';
     protected $primaryKeys = ['group_id', 'forum_id', 'auth_option_id', 'auth_role_id', 'auth_setting'];
```

Comment:
wonder if it's worth fixing this not actually being a unique key in the schema 🤔

Reply:
maybe another time ಠ_ಠ